docs/components/email/email-control-demo.js

```diff
@@ -0,0 +1,79 @@
+import Util from '../../_util';
+
+if (document.getElementById('vue-emailInputDemo')) {
+    new Vue({
+        el: '#vue-emailInputDemo',
+        data: {
+            hasAsterisk: false,
+            hasOptional: false,
+            hasHelpText : false,
+            hasErrorText: false,
+            isDisabled: false,
+            isRequired: false,
+            label: 'Email',
+            helpText: 'abc@abc.com',
+            errorText: 'Please enter valid email',
+            errorTextToDisplay: false,
+            helpTextToDisplay: false,
+            displayTrue :true,
+            displayFalse: false,
+        },
+        computed: {
+            attrDisabled: function () {
+                return (this.isDisabled ? 'disabled' : '');
+            },
+            attrRequired: function () {
+                return (this.isRequired ? 'required' : '');
+            },
+            lblClasses: function () {
+                let classes = [];
+
+                if (this.hasAsterisk) {
+                    classes.push('hxRequired');
+                }
+
+                if (this.hasOptional) {
+                    classes.push('hxOptional');
+                }
+
+                if (this.hasHelpText) {
+                    classes.push('hxHelpText');
+                    this.helpTextToDisplay = this.displayTrue;
+                } else {
+                    this.helpTextToDisplay = this.displayFalse;
+                }
+
+                if (this.hasErrorText) {
+                    classes.push('hxErrorText');
+                    this.errorTextToDisplay = this.displayTrue;
+                } else {
+                    this.errorTextToDisplay = this.displayFalse;
+                }
+
+                let classNames = classes.join(' ');
+
+                return (classNames === '' ? '' : `class="${classNames}"`);
+            },
+            snippet: function () {
+                return Util.snippet(`
+                    <hx-email-control>
+                      <input
+                        id="emailDemo"
+                        type="email"
+                        ${this.attrDisabled}
+                        ${this.attrRequired}
+                      />
+                      <label
+                        for="emailDemo"
+                        ${this.lblClasses}
+                      >
+                        ${this.label}
+                      </label>
+                      <p>${this.helpText}</p>
+                      <p>${this.errorText}</p>
```

Comment:
#### For the demo snippet, please append the `hxErrorText` class to this `<p>`.

<img width="805" alt="Screen Shot 2020-03-25 at 4 32 56 PM" src="https://user-images.githubusercontent.com/10120600/77594519-ba82c880-6ec4-11ea-8473-e12285882516.png">

Reply:
Changes have been done, please review.